interchaintests/fresh/baseline1_test.go

```diff
@@ -0,0 +1,26 @@
+package fresh_test
+
+import (
+	"testing"
+
+	"github.com/hyphacoop/cosmos-release-testing/interchaintests/fresh"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBaseline1(t *testing.T) {
```

Comment:
Can you make the job name a bit more verbose? Something like "TestBaseline1" -> "TestBaseline1-upgrade"